java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -67,9 +72,9 @@ public static String sanitizeString(String str) {
     }
 
     public static AbstractJSONSchema2Pojo fromJsonSchema(
-            String key, JSONSchemaProps prop, String prefix, String suffix) {
+            String key, JSONSchemaProps prop, String parentPkg, String prefix) {
```

Comment:
Maybe it's just me but here again, I'd name `prefix` as `classPrefix`, or something like that. WDYT?

Reply:
`classPrefix`